### PR TITLE
Fix some options don't apply (Android)

### PR DESCRIPTION
**Pull request by muv26**
For example, on Android, the option `:network-caching=300` had no effect when playing an RTSP stream.

### CODE REVIEW

**Comment by mrvancor** [comment]
> Hey @mrvancor, thanks for your contribution. Do you mind incrementing the version and adding a changelog so we can quickly release it?

Done.

File: flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/FlutterVlcPlayer.java [hunk 1]
```diff
@@ -353,6 +355,7 @@ void setStreamUrl(String url, boolean isAssetUrl, boolean autoPlay, long hwAcc)
                 media.addOption(":no-mediacodec-dr");
                 media.addOption(":no-omxil-dr");
             }
+            options.forEach(option -> media.addOption(option));
```
Comment (kostov) [inline flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/FlutterVlcPlayer.java]: see: https://github.com/solid-software/flutter_vlc_player/issues/288#issuecomment-1046482032

this change leads vlc-player doesn't work on Android 6
fix is change this line to orthodox for:

```
if (options != null) {
    for (String option: options) // works everywhere
        media.addOption(option);
}
```

Comment (illia-romanenko) [reply flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/FlutterVlcPlayer.java]: Feel free to create PR - I would be happy to merge it!

Comment (kostov) [reply flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/FlutterVlcPlayer.java]: https://github.com/solid-software/flutter_vlc_player/pull/301

Not sure properly decorated...

**Pull request merged**